README.md

```diff
@@ -27,42 +73,51 @@ The cloned repository files will be part of your actual repository, so anyone
 cloning this repo will have the files automatically without needing to use
 `git-subrepo` themselves.
 
-`ext` is just a convention, you can clone it into any directory.
+`external` is just a convention, you can clone it into any directory.
```

Comment:
All those "ext/external" changes look unrelated to your one single commit. Please split it into separate explicit git commits with according commit message.